### DISCUSSION TITLE
[Enterprise metadata bundle] Error in asset versions

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#11974] [Q&A / Support] [answered]
Hi. 

When I use Many-to-many-relation in Assets Metadata Class definition, with allow Assets checked and type document, and when I add some documents as metadata error is shown when I want to see asset version.

Screnshots:

![image](https://user-images.githubusercontent.com/17277953/165322852-c0a28355-d99f-459d-8238-7b41f3b3d118.png)

![image](https://user-images.githubusercontent.com/17277953/165322914-38ee40c9-cdd4-40f2-a053-68b748de6420.png)

in my project:
![image](https://user-images.githubusercontent.com/17277953/165322971-b51d6c0a-457e-42cf-a28e-b7fc3d66388d.png)

on pimcore demo is:
![image](https://user-images.githubusercontent.com/17277953/165323056-0ea13eed-b628-43d0-8b53-32c0641a6ae3.png)

Is that a bug ?

**Comment by fashxp** [comment, marked as answer]
It is a bug, will be released in release tomorrow
